fluent_pages/pagetypes/fluentpage/page_type_plugins.py

```diff
@@ -13,4 +14,5 @@ class FluentPagePlugin(FluentContentsPagePlugin):
     def get_render_template(self, request, fluentpage, **kwargs):
         # Allow subclasses to easily override it by specifying `render_template` after all.
         # The default, is to use the template_path from the layout object.
-        return self.render_template or fluentpage.layout.template_path
+        with switch_language(fluentpage):
```

Comment:
This seems unnecessary - `switch_language(obj)` changes the current Django language to match the object language that parler's `fluentpage.get_current_language()` would return. Since you only access model fields here, switching the language is not needed.